WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -15882,7 +15882,7 @@
 			files = (
 			);
 			inputFileListPaths = (
-				"$(SRCROOT)/secrets-manifest.xcfilelist",
+				"$(SRCROOT)/Credentials/secrets-manifest.xcfilelist",
```

Comment:
In a few other projects, I moved this file in the same folder as the script. What do you think of it?

Ideally, product developers should not need to interact with this file, so I think it can live outside the Xcode project.

Just an idea, though, I don't have a strong opinion about it.

Reply:
Yep makes more sense to have it near the script I agree 👍